ui/.dockerignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
May also want to add `swagger-codegen-cli.jar`

Reply:
I changed README instructions to put swagger-codegen-cli.jar in ~, since it doesn't make to have a separate jar for each repo.